documentation/modules/rql/pages/user-guide.adoc

```diff
@@ -189,30 +209,36 @@ Escape-char is backslash `\`, possible escape sequences are
 `\"`:: Double Quote
 `\f`:: Formfeed - *DO NOT USE* _this might be removed in future_
 `\b`:: Backspace - *DO NOT USE* _this might be removed in future_
+//TODO ? What do they do, why would I add them to a query?
```

Comment:
Remove the example which will be removed :)

Reply:
Done -- https://github.com/bci-oss/semanticstack-rql/pull/69/commits/1ebe646bb30b87ff32516e5b43cf6d3375910656